submissions/Ant-C-tech/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,466 @@
+      href="img/favicon/favicon-96x96.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="img/favicon/favicon-16x16.png"
+    />
+    <link rel="manifest" href="img/favicon/manifest.json" />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta
+      name="msapplication-TileImage"
+      content="img/favicon/ms-icon-144x144.png"
+    />
+    <meta name="theme-color" content="#ffffff" />
+    <!-- /Favicon -->
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;700&family=Rubik&display=swap"
+      rel="stylesheet"
+    />
+    <!-- /Fonts -->
+
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/app.css" />
+
+    <title>Cinema Application</title>
+  </head>
+
+  <body>
+    <video preload="auto" autoplay muted loop poster="img/video_poster.jpg" class="trailer">
+      <source
+        src="video/trailer.mp4"
+        type="video/mp4"
+      />
+    </video>
+
+    <main>
+      <section class="posterSection">
+        <div class="textContent">
+          <h2 class="textContent__title">Vivarium</h2>
+          <p class="textContent__description">
+            On their search for the perfect home, Gemma (Imogen Poots) and Tom
+            (Jesse Eisenberg) visit a new house in a labyrinthine suburban
+            neighbourhood. When they attempt to leave, each road mysteriously
+            takes them back to where they started, leading them on a
+            mind-bending trip, trapped in a surreal nightmare.
+          </p>
+          <a href="#" class="btn-readMore">&#9660;</a>
+        </div>
+
+        <div class="badge badge-rating">
+            <p>IMBd<br>5.8</p>
+        </div>
+
+        <div class="badge badge-data">
+            <p>From<br>01 March<br>1h 39m</p>
+        </div>
+
+        <div class="mask"></div>
+
+        <img
+          class="posterSection__img"
+          src="img/poster.jpg"
+          alt="vivarium poster"
+        />
+      </section>
+
+      <section class="controlSection">
+        <nav class="movieType">
+          <ul class="movieType__list">
+            <li class="movieType__listItem movieType__listItem-disabled">
+              <a class="movieType__listItemLink movieType__listItemLink-disabled" href="#">General</a>
+            </li>
+            <li class="movieType__listItem">
+              <a class="movieType__listItemLink" href="#">Digital 2D</a>
+            </li>
+            <li class="movieType__listItem movieType__listItem-active">
+              <a class="movieType__listItemLink movieType__listItemLink-active" href="#">IMAX 3D</a>
+            </li>
+            <li class="movieType__listItem">
+              <a class="movieType__listItemLink" href="#">Cinema 4Dx</a>
+            </li>
+          </ul>
+        </nav>
+
+        <nav class="movieDate">
```

Comment:
Why you use `<nav>` for this section?

Reply:
I have some doubts about correct tags for these elements but using this resource http://yoksel.github.io/easy-markup/first-steps/ according your advice I have thought that they have to be section-tags because I consider that for this page of application "controlSection"-element acts a part of "main"-element. I hope that I did it properly.